frontends.go

```diff
@@ -84,7 +84,8 @@ func HttpHandlerLogin(config *ConfigHTTP, central *Central, w http.ResponseWrite
 		return
 	}
 
-	if sessionkey, token, err := central.Login(identity, password); err != nil {
+	clientIp := r.Header.Get("X-Forwarded-For")
```

Comment:
Try first for `X-Forwarded-For`. If that is not present, then fall back to `RemoteAddr`